runtime/doc/deprecated.txt

```diff
@@ -142,6 +142,8 @@ LSP FUNCTIONS
 - *vim.lsp.get_active_clients()*		Use |vim.lsp.get_clients()|
 - *vim.lsp.for_each_buffer_client()*		Use |vim.lsp.get_clients()|
 - *vim.lsp.util.trim_empty_lines()*		Use |vim.split()| with `trimempty` instead.
+- *vim.lsp.util.try_trim_markdown_code_blocks()*
```

Comment:
Thoughts on not mentioning alternatives here?

Reply:
No objection.